test/integration/batch_inviting_users_test.rb

```diff
@@ -22,7 +22,7 @@ class BatchInvitingUsersTest < ActionDispatch::IntegrationTest
       invited_user = User.find_by_email("fred@example.com")
       assert_not_nil invited_user
       assert invited_user.has_access_to?(application)
-      assert_equal "noreply-signon-development@digital.cabinet-office.gov.uk", last_email.from[0]
+      assert_match /noreply-signon-development@.+\.gov\.uk/, last_email.from[0]
```

Comment:
I don't think we should be making this 'vague' in the integration tests. I appreciate that this approach is used in the unit tests but they are testing parts of the email address rather than the whole. Could we add another test that explicitly sets a different locale in the test and asserts that the locale config is being respected? Would that work for your use case @dwhenry ?